src/baseline/expr.rs

```diff
@@ -778,16 +778,37 @@ where
         if let Some(intrinsic) = self.intrinsic(e.id) {
             self.emit_intrinsic_bin(&e.lhs, &e.rhs, dest, intrinsic, Some(e.op));
         } else if e.op == BinOp::Cmp(CmpOp::Is) || e.op == BinOp::Cmp(CmpOp::IsNot) {
```

Comment:
Could you please move this code-block into a new function `emit_bin_is`?

Reply:
Sure!